atflatcontrols/atlinklabel.pas

```diff
@@ -5,30 +5,58 @@
 
 unit ATLinkLabel;
 
+{$ifdef FPC}
 {$mode objfpc}{$H+}
+{$else}
+  {$define delphi}
+  {$define windows}
+  {$ifdef VER150} //Delphi 7
+    {$define WIDE}
+  {$endif}
+{$endif}
 
```

Comment:
this unit needs WIDE, VER150?

Reply:
Not sure, I just added it since it displays text. It will work removed. Should I remove it?

Reply:
Will work removed ?? So this block don’t help? Remove it

Reply:
It looks like this now:
{$ifdef FPC}
{$mode objfpc}{$H+}
{$else}
  {$define delphi}
  {$define windows}
{$endif}
If I remove anymore it errors.